### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Marielvacacruz**
could I get my tables/models/references/associations reviewed please?

### CODE REVIEW

File: authenticate-me/backend/db/models/user.js [hunk 1]
```diff
@@ -43,29 +43,54 @@ module.exports = (sequelize, DataTypes) => {
...
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: DataTypes.INTEGER
+    },
     username:{
       allowNull: false,
-      type: DataTypes.STRING,
+      type: DataTypes.STRING(30),
```
Comment (philling83) [inline authenticate-me/backend/db/models/user.js]: (30) is not needed as the len validation provides the range

File: authenticate-me/backend/db/models/venue.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+module.exports = (sequelize, DataTypes) => {
+  class Venue extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      Venue.hasMany(
+        models.Event, { foreignKey: 'venueId'} //do I need an onDelete Cascade? if a venue is deleted does the event
```
Comment (philling83) [inline authenticate-me/backend/db/models/venue.js]: Yeah I would say having onDelete Cascade makes sense here

File: authenticate-me/backend/db/migrations/20220517224555-create-event.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      venueId: {
+        references: {
+          model: 'Venues',
+          key: 'id'
+        },
+        // onDelete: 'cascade', <-- is this needed?
```
Comment (WJVincent) [inline authenticate-me/backend/db/migrations/20220517224555-create-event.js]: Only if you want the event to be deleted if the venue is deleted

File: authenticate-me/package.json [hunk 4]
```diff
@@ -20,5 +20,8 @@
   "bugs": {
     "url": "https://github.com/Marielvacacruz/meetup-portfolio-project/issues"
   },
-  "homepage": "https://github.com/Marielvacacruz/meetup-portfolio-project#readme"
+  "homepage": "https://github.com/Marielvacacruz/meetup-portfolio-project#readme",
+  "dependencies": {
+    "pg": "^8.7.3"
```
Comment (WJVincent) [inline authenticate-me/package.json:25]: This should be a package in the backend/package.json rather than in the root package.json

**Pull request closed without merging**